tests/integration/kubernetes/gha-run.sh

```diff
@@ -175,6 +175,7 @@ function get_nodes_and_pods_info() {
 	echo "::endgroup::"
     fi
     kubectl debug $(kubectl get nodes -o name) -it --image=quay.io/kata-containers/kata-debug:latest
+    kubectl get pods -o name | grep node-debugger | xargs kubectl delete
```

Comment:
@fidencio @sprt   We could have a label for the kata-debug pods then we could omit the `grep` and `xargs` for a cleaner 
```
kubectl delete pods -l name=kata-debug-<whatever>
```

Reply:
I agree it's not super clean, but you can't pass a label to `kubectl debug`, and getting the exact name of a debug pod would also look ugly.